tools/ci_build/github/linux/run_dockerbuild.sh

```diff
@@ -69,6 +70,27 @@ else
             docker build -t "onnxruntime-$IMAGE" --build-arg BUILD_USER=onnxruntimedev --build-arg BUILD_UID=$(id -u) --build-arg OS_VERSION=16.04 --build-arg PYTHON_VERSION=${PYTHON_VER} -f Dockerfile.ubuntu .
         fi
     fi
+else
+    if [ $BUILD_DEVICE = "gpu" ]; then
```

Comment:
do we have ubuntu18 image for cuda ep?

Reply:
can we refactor this a bit to reduce duplication?
this gpu block here is exactly the same as the gpu block above.
same for openvino, if you want to support for both ubuntu16 and 18.
the only difference is ubuntu18.04 vs ubuntu16.04, which you can use $BUILD_OS variable.